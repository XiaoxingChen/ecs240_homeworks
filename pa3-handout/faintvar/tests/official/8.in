c Multiple loops
p 3 6 7 1 6
c bb1: v1 = c;
b 1 1
e 1 2
c bb2: ;
b 2
e 2 3
e 2 4
c bb3: v2 = v1;
b 3 2 1
e 3 2
c bb4: ;
b 4
e 4 5
e 4 6
c bb5: v1 = v2;
b 5 1 2
e 5 4
c bb6: print(v1);
b 6 0 1
