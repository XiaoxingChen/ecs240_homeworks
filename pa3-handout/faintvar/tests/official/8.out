fvin 1 1 3
fvin 2 3
fvin 3 2 3
fvin 4 3
fvin 5 1 3
fvin 6 2 3
